packages/react-app/src/App.jsx

```diff
@@ -23,7 +23,7 @@ function App() {
       </Header>
       <Body>
         <ContractProvider>
-          <FinancialRole />
+          <RoleComponents />
```

Comment:
I suggest something more meaningful

```suggestion
          <RolePageSwitcher />
```